test/travis_ci/docker/ubuntu_18_04/install_ubuntu.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -v
+
+# setup repo with recent gcc versions
+add-apt-repository -y ppa:ubuntu-toolchain-r/test
+add-apt-repository -y ppa:teward/swig3.0
+#add-apt-repository -y ppa:george-edison55/cmake-3.x
+
+# suck in package lists
+apt-get update -qq
+
+# install deps
+# use PIP for Python packages
+apt-get install -qq -y git-core gcc-5 g++-5 gfortran-5 cmake swig3.0 \
```

Comment:
more cruft from 14.04. please get rid of version numbers  on packages. should not be necessary as 18.04 has new enough versions, where 14.04 did not (eg. gcc 4 didn't have full c++11 implementation hence, we needed 5)